README.md

```diff
@@ -1,15 +1,26 @@
-
-## Models
-
-This package contains transformation models, designed to work simultaneously with our [Marketo source package](https://github.com/fivetran/dbt_marketo_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below. Intermediate models are used to create these output models.
-
-| **model**                | **description**                                                                                                                                |
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_marketo/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_,<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+
+# Marketo Transformation dbt Package ([docs](https://fivetran.github.io/dbt_marketo/))
+# 📣 What does this dbt package do?
+- Produces modeled tables that leverage Marketo data from [Fivetran's connector](https://fivetran.com/docs/applications/marketo) in the format described by [this ERD](https://fivetran.com/docs/applications/marketo#schema) and builds off the output of our [Marketo source package](https://github.com/fivetran/dbt_marketo_source).
+- Enables you to better understand your Marketo email performance and how your leads change over time. The output includes models with enriched email metrics for leads, programs, email templates, and campaigns. It also includes a lead history table that shows the state of leads on every day, for a set of columns that you define.
+- Generates a comprehensive data dictionary of your source and modeled Marketo data through the [dbt docs site](https://fivetran.github.io/dbt_marketo/).
+
+The following table provides a detailed list of all models materialized within this package by default.
+
+> TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_marketo/).
+
+| **Model**                | **Description**                                                                                                                                |
 | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
 | [marketo__campaigns](https://github.com/fivetran/dbt_marketo/blob/main/models/marketo__campaigns.sql)       | Each record represents a Marketo campaign, enriched with metrics about email performance.                                                      |
```

Comment:
For each of the model references, can you update them to reference the docs site instead of the sql files.

Reply:
Done. Just a note that the links won't work now until after merging with main.